src/controllers/connection/__tests__/index.test.ts

```diff
@@ -0,0 +1,34 @@
+import { describe, it } from 'mocha'
+import sinon from 'sinon'
+
+import { toHTMLString, withMocks } from './helpers'
+
+import { ConnectionController } from '../index.js'
+
+describe('ConnectionController', () => {
+  let expect: Chai.ExpectStatic
+  before(async () => {
+    expect = (await import('chai')).expect
```

Comment:
"thought will be way uglier implementation. Wonder if it's possible to preload this for all test suites"

Reply:
I have tried to no end to make this load as a global. I just cannot get `tsimp` or `tsx` to see it. This is the best I've managed and it's ugly. We can simplify this once https://github.com/mochajs/mocha/issues/5139 is fixed